ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/DecoderOptions.java

```diff
@@ -37,6 +38,34 @@
 
   void setRedisUri(ValueProvider<String> value);
 
+  @Description("If set to true, enable decryption of Pioneer payloads.")
+  @Default.Boolean(false)
+  Boolean getPioneerEnabled();
+
+  void setPioneerEnabled(Boolean value);
+
+  @Description("Path (local or gs://) to JSON array of metadata entries enumerating encrypted"
+      + " private keys, Cloud KMS resource ids for decrypting those keys, and their corresponding"
+      + " document namespaces; leave null to disable.")
+  ValueProvider<String> getPioneerMetadataLocation();
+
+  void setPioneerMetadataLocation(ValueProvider<String> value);
+
+  @Description("If set to true, assume that all private keys are encrypted with the associated"
+      + " KMS resourceId. Otherwise ignore KMS and assume all private keys are stored in plaintext."
+      + " This may be used for debugging.")
+  @Default.Boolean(true)
```

Comment:
Last I checked, default values were ignored for value providers in practice, although this is not documented. To provide a default value for a ValueProvider, you have to check at time of use and inject a default value there. We should remove this annotation as it has no effect.

Reply:
I've added a test to check for the default behavior, it seems to work as expected. The ValueProvider does set the value to true.